### ISSUE TITLE
Problem with displaying time with milisecond.

### ISSUE THREAD

**Issue by alextran1502** [leeoniya/uPlot#280]
Hello @leeoniya ,

Thank you for a wonderful chart library, I love it so much.

I am having a problem with displaying my data with ms interval. Here is my data format look like
![image](https://user-images.githubusercontent.com/27055614/89050344-3d56dc80-d318-11ea-8de3-940c5fd81ba4.png)

And I have `scales.x.time: true` for my chart option. 

Somehow the chart doesn't show anything.

![image](https://user-images.githubusercontent.com/27055614/89050439-5fe8f580-d318-11ea-8beb-8b6f13f42a23.png)

Can you give me some pointer on how to resolve this issue?

**Comment by leeoniya** [comment]
you have your data sorted backwards. thr timestamps should be sorted ASC.

not sure if that's the exact cause though without seeing thw whole setup.

**Comment by alextran1502** [comment]
@leeoniya yep... I have it sorted backward. Thank you very much! that solves the problem
